driver-core/src/test/resources/crud/write/updateWithPipelines.json

```diff
@@ -0,0 +1,239 @@
+{
```

Comment:
Until all tests are moved over to v2 I think we should separate out the v1 and v2 tests.  Perhaps create a crud/v1 and crud/v2 directory? We need some way that makes it easier to sync the tests and understand what other tests need porting.